firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -147,8 +152,8 @@ public class FirebaseCrashlytics {
 
     Logger.getLogger().v("Installer package name is: " + appData.installerPackageName);
 
-    final ExecutorService threadPoolExecutor =
```

Comment:
this might need to refactor to using the same sequential queue as the one for user actions.